db/pinned_iterators_manager.h

```diff
@@ -16,7 +16,7 @@ namespace rocksdb {
 // PinnedIteratorsManager will be notified whenever we need to pin an Iterator
 // and it will be responsible for deleting pinned Iterators when they are
 // not needed anymore.
-class PinnedIteratorsManager {
+class PinnedIteratorsManager : public Cleanable {
```

Comment:
now the cleanup code will be called in the destructor, I think we must change that so that it's called in `ReleasePinnedData()`